src/styles/ThemeStyles.ts

```diff
@@ -0,0 +1,12 @@
+import { createMuiTheme } from "@material-ui/core/styles";
```

Comment:
The theming conventions that you added look great :') 
Moved it to a separate file so that themed styling can be established from App.tsx and can be reflected throughout the screens!